src/lsm/manifest_level.zig

```diff
@@ -38,6 +38,22 @@ pub fn ManifestLevelType(
 
         pub const Tables = SegmentedArray(TableInfo, NodePool, table_count_max, .{});
 
+        pub const TableInfoReference = struct { table_info: *TableInfo, generation: u64 };
```

Comment:
Either `u64` or `u32` is fine for `generation`, but be consistent. (`ManifestLevel.generation` is currently `u32`, `TableInfoReference.generation` is `u64`).